cloudfront/wellcomelibrary.org/edge-lambda/src/api.ts

```diff
@@ -0,0 +1,59 @@
+import axios from "axios";
+import {Work, CatalogueResultsList} from "./catalogue";
+
+export type GetWorkResult = Work | Error
+
+const apiBasePath = 'https://api.wellcomecollection.org/catalogue/v2'
+
+type ApiQuery = {
+    query: string
+    include: Array<string>
+}
+
+async function* apiQuery(query: ApiQuery): AsyncGenerator<Work, void, void> {
+    const url = apiBasePath + '/works';
+    const apiResult = await axios.get(url, {params: query}  );
```

Comment:
If you call the argument `params` you can just have
```suggestion
    const apiResult = await axios.get(url, { params });
```

I personally really like to keep arguments between methods the same to avoid cognitive load on what's going on. With limits obviously.